pkg/reconciler/tenancy/clusterworkspace/clusterworkspace_controller.go

```diff
@@ -53,20 +53,20 @@ const (
 func NewController(
 	kcpClusterClient kcpclient.Interface,
 	workspaceInformer tenancyinformer.ClusterWorkspaceInformer,
-	rootWorkspaceShardInformer tenancyinformer.ClusterWorkspaceShardInformer,
+	clusterWorkspaceShardInformer tenancyinformer.ClusterWorkspaceShardInformer,
```

Comment:
This seems like an unrelated change?

Reply:
no, we had `rootWorkspaceShardInformer` in the server for reasons that must have disappeared. I removed the `root*` variant of those informers. Hence, also here.